frontend/packages/client/src/components/Modals/index.js

```diff
@@ -0,0 +1,6 @@
+export { default as Modal } from './Modal';
+export { default as Error } from './Error';
+export { default as VoteConfirmation } from './VoteConfirmation';
+export { default as CastingVote } from './CastingVote';
+export { default as VoteConfirmed } from './VoteConfirmed';
+export { default as CancelProposal } from './CancelProposal';
```

Comment:
Modal is the building block for other Modals then other files can reuse styles and it's easier for getting the component idea